### PR TITLE
refactor: Miniminer package linearization followups

### PR DESCRIPTION
**Pull request by kevkevinpal**
### Motivation
In https://github.com/bitcoin/bitcoin/pull/28762 there were some post merge comments which are being addressed in this PR with the following commits

### [8d4c46f](https://github.com/kevkevinpal/bitcoin/pull/5/commits/8d4c46f54d10fb67d20d7a9a6afa37ecfd2bdc18) Reorganizing `MiniMinerMempoolEntry` to match the order we have elsewhere
* https://github.com/bitcoin/bitcoin/pull/28762#discussion_r1381775670

### [7505ec2](https://github.com/kevkevinpal/bitcoin/pull/5/commits/7505ec2054e5918068e1f18f290c8d5e9e4c2590) Renaming `cached_descendants` to `descendants` for simpler variable naming
* https://github.com/bitcoin/bitcoin/pull/28762#discussion_r1381819567

### [b21f2f2](https://github.com/kevkevinpal/bitcoin/pull/5/commits/b21f2f2f5567dc16953df13d4423a315c2a040e1) Code comment modifications to be more accurate to what is actually happening 
* https://github.com/bitcoin/bitcoin/pull/28762#discussion_r1381902909 and
* https://github.com/bitcoin/bitcoin/pull/28762#discussion_r1382002278 and
* https://github.com/bitcoin/bitcoin/pull/28762#discussion_r1383041819

### CODE REVIEW

**Comment by glozow** [comment]
This diff appears to have added new lines with trailing whitespace.
The following changes were suspected:

diff --git a/src/test/miniminer_tests.cpp b/src/test/miniminer_tests.cpp
@@ -668,7 +667,7 @@ BOOST_FIXTURE_TEST_CASE(manual_ctor, TestChain100Setup)
+        miniminer_info.emplace_back(grandparent_zero_fee,          /*vsize_self=*/tx_vsize,/*vsize_ancestor=*/tx_vsize, /*fee_self=*/0,/*fee_ancestor=*/0);      
@@ -696 +695 @@ BOOST_FIXTURE_TEST_CASE(manual_ctor, TestChain100Setup)
+        // CPFP low + double low

**Comment by kevkevinpal** [comment]
> This diff appears to have added new lines with trailing whitespace. The following changes were suspected:
> 
> diff --git a/src/test/miniminer_tests.cpp b/src/test/miniminer_tests.cpp @@ -668,7 +667,7 @@ BOOST_FIXTURE_TEST_CASE(manual_ctor, TestChain100Setup)
> 
>     * ```
>          miniminer_info.emplace_back(grandparent_zero_fee,          /*vsize_self=*/tx_vsize,/*vsize_ancestor=*/tx_vsize, /*fee_self=*/0,/*fee_ancestor=*/0);      
>       ```
> 
> 
> @@ -696 +695 @@ BOOST_FIXTURE_TEST_CASE(manual_ctor, TestChain100Setup)
> 
>     * ```
>          // CPFP low + double low 
>       ```

Thanks! fixed in [43423fd](https://github.com/bitcoin/bitcoin/pull/28808/commits/43423fd834df03c2eb88fd5d7007487816865bd6) and [89c1468](https://github.com/bitcoin/bitcoin/pull/28808/commits/89c146874175f8481c59436813ccf52a177a8664)

File: src/node/mini_miner.h [hunk 1]
```diff
@@ -138,7 +138,7 @@ class MiniMiner
     MiniMiner(const CTxMemPool& mempool, const std::vector<COutPoint>& outpoints);
 
     /** Constructor in which the MiniMinerMempoolEntry entries have been constructed manually,
```
Comment (murchandamus) [inline src/node/mini_miner.h:140]: Nit: First sentence ends in a comma now.

**Review by murchandamus — COMMENTED** [review]
ACK 89c146874175f8481c59436813ccf52a177a8664

 This does address the comments I left on the prior PR https://github.com/bitcoin/bitcoin/pull/28762, and the conflicting PR appears to still be a draft. Looks cleaner and easier to understand to me, but also not particularly urgent.

File: src/node/mini_miner.h (hunk 1 above)
Comment (kevkevinpal) [reply src/node/mini_miner.h]: fixed in [b4b01d3](https://github.com/bitcoin/bitcoin/pull/28808/commits/b4b01d3fb42e7b688d97b75f57cfe18cfca6d943)

**Review by murchandamus — COMMENTED** [review]
Trivial fix of my nit and improved formatting of the corresponding comment.

reACK b4b01d3fb42e7b688d97b75f57cfe18cfca6d943

**Review by theStack — APPROVED** [review]
LGTM ACK b4b01d3fb42e7b688d97b75f57cfe18cfca6d943

**Comment by glozow** [comment]
@kevkevinpal (nit) in the future, it's best to delete the help text before you open the PR as it gets pulled into the merge commit

**Pull request merged**